solution_runner/utils.py

```diff
@@ -0,0 +1,17 @@
+import importlib
+from pathlib import Path
+from types import ModuleType
+
+import consts
+
+
+def get_module_from_filepath(filepath: Path, directory_relative_to: Path | str = Path('.')) -> ModuleType:
+    """
+    :param filepath: file path to import
+    :param directory_relative_to: directory to compute relative path to from `file_path`, default is current directory
+    :return: imported module from `file_path`
+    """
+    module_name = str(filepath.relative_to(directory_relative_to.absolute()).with_suffix(''))
```

Comment:
Why is `.with_suffix('')` necessary?

Reply:
To get the absolute path to the file without the extension.